components/Navbar.js

```diff
@@ -40,14 +40,14 @@ const StyledNavbar = styled.nav`
   left: 0;
   right: 0;
   z-index: 0;
-  height: 70px;
+  height: 60px;
 `;
 
 const StyledLinkNavbar = styled.a`
   ${({ theme }) => css`
     font-size: ${theme.fonts.fontSizeButton};
   `}
-  color: ${({ isActive }) => (isActive ? '#2AFE41' : 'white')};
+  color: ${({ isActive }) => (isActive ? '#D25CFF' : 'white')};
```

Comment:
A fun little idea: your buttons have a sort of 'made of wood' aesthetic. Perhaps you'd prefer a muted brown color for your inactive state, and a white, or even bronze-like color for your active state. The pink is a little jarring, in my very subjective opinion. (And I clearly love pink otherwise.)